utils/build.ps1

```diff
@@ -3455,11 +3462,11 @@ if ($IncludeNoAsserts) {
   Build-NoAssertsToolchain
 }
 
-if (-not $SkipBuild) {
+if (-not $SkipBuild -and -not(Compare-Object $Test @("lldb"))) {
```

Comment:
This absolutely should happen irrespective - we are changing the memory allocator on the toolchain and the tests should be running with that memory allocator.